src/checkbox/ux-checkbox.ts

```diff
@@ -32,7 +32,13 @@ export class UxCheckbox implements Themable {
   private ripple: PaperRipple | null = null;
 
   public get isDisabled() {
```

Comment:
If we are going to bind to this property, we should add a `@computedFrom('disabled')` decorator to this property.